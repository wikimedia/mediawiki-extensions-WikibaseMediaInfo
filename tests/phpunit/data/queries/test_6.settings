{
	"title": "keyword only search",
	"term": "hastemplate:foo",
	"entityIdsForTerm": [],
	"defaultProperties": [
		"P999"
	],
	"userLanguage": "de",
	"fallbackLangs": [],
	"stemmingSettings": {},
	"settings": {
		"caption": 1.0,
		"caption-fallback-discount": 0.1,
		"all": 0.01,
		"all.plain": 0.02,
		"category": 0.03,
		"title": 0.04,
		"redirect.title": 0.05,
		"text": 0.06,
		"auxiliary_text": 0.07,
		"file_text": 0.08,
		"non-file_namespace_boost": 20.0
	}
}
