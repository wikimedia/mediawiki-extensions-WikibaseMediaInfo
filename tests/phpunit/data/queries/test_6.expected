{
    "bool": {
        "must": [
            {
                "match_all": {}
            }
        ],
        "filter": [
            {
                "match": {
                    "template": {
                        "query": "Template:Foo"
                    }
                }
            }
        ]
    }
}